tao_compiler/mlir/xla/ral/context/stream_executor_based_impl.cc

```diff
@@ -267,6 +267,14 @@ struct RalGemmState : public Context::Resource {
   std::map<GemmTuningCacheKey, se::blas::AlgorithmType> gemm_tuning_cache;
 };
 
+bool blade_gemm(void* s, bool fp16_in, bool fp16_out,
+                const void* a, int as0, int as1, bool tpa,
+                const void* b, int bs0, int bs1, bool tpb,
+                void* c, int cs0, int cs1
+               ) {
+  return false;
```

Comment:
It's a WIP PR?

Reply:
Obviously...:)

Reply:
No, it not a WIP PR. The implementation of blade_gemm will be kept internal.

Reply:
Since we have another way of keeping implementation of blade_gemm internal with macro '#ifdef PLATFORM_ALIBABA', I will have another PR to do this later.